src/apollo/config.js

```diff
@@ -7,24 +7,27 @@ import {
 } from '@apollo/client';
 import { getMainDefinition } from '@apollo/client/utilities';
 import { WebSocketLink } from '@apollo/client/link/ws';
+import { replaceMatches } from '../utils';
 
-const wsHost =
-  process.env.NODE_ENV === 'development'
-    ? 'localhost:4142'
-    : window.location.host;
+const { host, pathname, protocol } = window.location;
+const sanitizedPathname = replaceMatches(pathname, {
+  'experiment-tracking': '',
+});
+
+const wsHost = process.env.NODE_ENV === 'development' ? 'localhost:4142' : host;
 
-const wsProtocol = window.location.protocol === 'https:' ? 'wss' : 'ws';
+const wsProtocol = protocol === 'https:' ? 'wss' : 'ws';
```

Comment:
```suggestion
const wsProtocol = protocol === 'https:' ? 'wss:' : 'ws:';
```
Just to be consistent about what "protcol" refers to.